store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9624,4 +9624,9 @@ TODO: delete them permanently from here
   </desc>
 </attr>
 
+<attr id="3022" name="zimbraAdminAuthFallbackToLocal" type="boolean" cardinality="single" optionalIn="globalConfig,domain" since="8.8.6">
```

Comment:
I think attributed description has to be more specific to describe the logic of the code being added to LdapProvisioning.java, which is that if zimbraAuthFallbackToLocal is set to TRUE, then any account (admin or non admin) will fall back to local authentication, but if zimbraAuthFallbackToLocal is set to FALSE, then it is possible to allow only admin accounts to fall back to local authentication by setting this new attribute to TRUE, which is also the default behavior. If the desired outcome is to not allow falling back to local authentication for any accounts, then it is necessary to set both attributes to FALSE.